### DISCUSSION TITLE
Boost Move Hub communication

### DISCUSSION THREAD

**Discussion by 74Andre** [pybricks/support#531] [Q&A]
Gentlepeople,

I'm considering to install Pybrick for my kids on their Boost to start coding python in an interactive playful way.

As i understand, the Move hub has limited flash and there for not all functionality is available.
Is there such overview what is exactly supported (per hub type)?

For the Boost Move hub, is there a way to send command to the hub?

Is the print() (put_char?) streaming data from the hub over the BLE into the host PC? 
Is there also a get_char alike method  ? 

Keep up the good stuff!

**Comment by laurensvalk** [comment]
The [latest version of the documentation](https://docs.pybricks.com/en/latest/index.html) has a **compatibility** button near most features, where you can see which hubs are supported.

For example, in case of the [umath module](https://docs.pybricks.com/en/latest/micropython/umath.html):

![image](https://user-images.githubusercontent.com/12326241/142425984-1bf0d33a-617e-429c-b7b9-ebdd83375420.png)

> Is the print() (put_char?) streaming data from the hub over the BLE into the host PC?
Is there also a get_char alike method ?

`print` is available, but something like `get_char` is not, unfortunately. As you already noticed, this hub does not have much storage space for such extras.
